src/controllers/blocks/BlocksController.ts

```diff
@@ -169,4 +166,77 @@ export default class BlocksController extends AbstractController<BlocksService>
 			await this.service.fetchBlock(hash, options)
 		);
 	};
+
+	/**
+	 * Return the Header of a block when a block number is queried
+	 *
+	 * @param req Express Request
+	 * @param res Express Response
+	 */
+	private getBlockByIdSummary: RequestHandler<INumberParam> = async (
```

Comment:
```suggestion
	private getBlockSummaryById: RequestHandler<INumberParam> = async (
```

Reply:
or `getBlockHeaderById`

Reply:
I like the header notion here! for now I am going to switch it all over to Header, I think it makes more sense since that is the exact Type we are returning.